assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/styles.scss

```diff
@@ -3,14 +3,21 @@
  * © 2020 Event Espresso
  * ------------------------------------------------------------------------- */
 
-.ticket-assignment-manager.ee-rspnsv-table__outer_wrapper {
-	.ee-rspnsv-table {
+@import 'application/ui/styles/mixins/breakpoints.scss';
+
+.ticket-assignment-manager.ticket-assignment-manager.ee-rspnsv-table__outer_wrapper {
+	.ee-rspnsv-table.ee-rspnsv-table {
 		background: var(--ee-background-color);
 
 		&__inner-wrapper {
 			background: var(--ee-background-color);
 			margin: 0 0 var(--ee-margin-smaller) 300px;
-			width: calc(100% - 300px) !important;
+			width: calc(100% - 300px);
```

Comment:
👍 👍  I have a bad habit of falling back on this when I get stuck instead of spending the extra time to solve things properly